src/phases/move-effect-phase.ts

```diff
@@ -850,7 +860,7 @@ export class MoveEffectPhase extends PokemonPhase {
     }
 
     if (damage <= 0) {
-      return [result, isCritical];
+      return [result, isCritical, dmg];
```

Comment:
```suggestion
      return [result, isCritical, damage];
```
Pretty sure everything after the "damage" variable is set should use it and not the pre-calculation "dmg" variable.

Reply:
thanks, sloppy oversight

Reply:
I think in the future we should probably change "dmg" to "rawDamage" and "damage" to "finalDamage".